docs/binary.md

```diff
@@ -13,10 +13,19 @@ nav_order: 2
 ## Introduction
 
 ```yaml
-A Binary Number is made up of only 0s and 1s.
-Example: 110111
-
+Binary number system was invented by Gottfried Leibniz. As the word is prefixed with 'Bi' which is a
+Latin word and means 'two' in English. This brings us to the first two digits i.e., 0 and 1 which
+means that while counting in binary we cannot exceed 1. Infact all the numbers which we tend to
+represent are made up of only two digits i.e., 0 and 1 which is quite interesting.
+Let us show you the binary representation of a decimal number(The numbers which we use for counting
+i.e., from 0-9) in binary.
+Example:
+Decimal number :: 25
+Binary Number :: 11001
+
+Note ::
 There is no 2, 3, 4, 5, 6, 7, 8 or 9 in Binary!
+Also, each digit in binary representation is either 0 or 1.
```

Comment:
You have used this fact in the above paragraph. Please change this.

Reply:
Removed the redundant line